### ISSUE TITLE
Issue while logging in to Skoda connect via HASSIO

### ISSUE THREAD

**Issue by gourmet77** [lendy007/homeassistant-skodaconnect#110]
I have an issue when connecting through the configuration in HASISO - the process stops with loading animation after inputing my email/password. No further steps are following, as the animation remain in view.

When checking the log file, it states:

Login failed with error: The terms and conditions must be accepted first at "https://www.skoda-connect.com/"
2:11:10 PM – (ERROR) Skoda Connect (custom integration) - message first occurred at 2:00:42 PM and shows up 2 times
Login failed, the terms and conditions might have been updated and need to be accepted. Login to https://www.skoda-connect.com and accept the new terms before trying again
2:11:10 PM – (WARNING) /usr/local/lib/python3.9/site-packages/skodaconnect/connection.py - message first occurred at 2:00:42 PM and shows up 2 times

When logging into website of Skoda-connect.cz, no terms and conditions are asked for, I can browse through the options of my car without any problems.

Please advice.

**Comment by gourmet77** [comment]
Found the issue - SKODA CONNECT does not push new T&C throuch Chrome, but it does in Windows Edge. When using Edge the request to accept showed up. Once agreed =connection is working now.

**Comment by gourmet77** [comment]
Problem solved - issue of the browser.

**Comment by Farfar** [comment]
> 

Yes there's often problems with receiveing new T&C in Chrome, most often because of cached content. Clear site data or incognito window works sometimes. But logging in through different browser is often easier and more reliable.
